recipes/longhorn.yaml

```diff
@@ -0,0 +1,69 @@
+# Longhorn recipe
+# Description: Installs iscsi daemonset, longhorn and ingress-nginx
+# Tested on:
+# - EKS eks.5
+# - GKE 1.19.9-gke.1900
+# - Digital Ocean 1.21.2-do.2
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: longhorn
+    url: https://charts.longhorn.io
+    type: helm
+  namespace: "longhorn-system"
+  pre_install:
+  - steps:
+    - openssl version
+      # Recipe needs USER and PASSWORD environment variables for ingress basic auth
+    - ': "${USERNAME:?Variable USERNAME not set or empty}" : "${PASSWORD:?Variable PASSWORD not set or empty}"'
+    - curl -s https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/scripts/environment_check.sh | bash
+      # daemonset for iscsi service installation
+    - kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn-manager/master/deploy/prerequisite/longhorn-iscsi-installation.yaml
+  post_install:
+  - apps:
+    - ingress-nginx
+  - steps:
+    - echo ${USERNAME}:$(echo ${PASSWORD} | openssl passwd -stdin -apr1) >> /tmp/auth
+    - kubectl -n longhorn-system create secret generic basic-auth --from-file=/tmp/auth
```

Comment:
Instead of creating files, can we just set that in a variable and use `--from-literal` flag to create secret out of it